pages/Documents/Reporting/OutboundConfigurationAPI/overview.md

```diff
@@ -0,0 +1,400 @@
+| :------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :--------------- |
+| Content-Type  | Used to indicate the media type of the resource                                                                                                                                                                                                     | application/json |
+| Authorization | [OAuth 2.0](https://developers.liveperson.com/connector-api-send-api-authorization-and-authentication.html#get-appjwt) or [OAuth 1.0](https://developers.liveperson.com/retrieve-api-keys-create-a-new-api-key.html) (Section 8) or LE Bearer token |                  |
+
+**Response Example**
+
+200 Success
+
+outbound configuration by response example:
+```json
+[
+    {
+        "type": "sms-twilio",
+        "language": "en",
+        "outboundnumber": "+13333333333",
+        "messageComponents": [
+            {
+                "type": "body",
+                "content": "Hi this is an outbound message #1",
+                "variables": ["1"],
+                "optOut": "To stop receiving messages, reply STOP. For help, reply HELP."
+            }
+        ]
+    },
+    {
+        "type": "sms-twilio",
+        "language": "en",
+        "outboundnumber": "+142222222222",
+        "messageComponents": [
+            {
+                "type": "body",
+                "content": "Hi this is an outbound message #1",
+                "variables": ["1", "2"],
+                "optOut": "To stop receiving messages, reply STOP. For help, reply HELP."
+            }
+        ]
+    },
+    {
+        "type": "sms-twilio",
+        "language": "en",
+        "outboundnumber": "+1433333333333",
+        "messageComponents": [
+            {
+                "type": "body",
+                "content": "Hi this is an outbound message #3",
+                "variables": ["1"],
+                "optOut": "To stop receiving messages, reply STOP. For help, reply HELP."
+            }
+        ]
+    }
+]
+```
+
+### Data fields definitions
+
+Outbound Reporting API provides the following data fields. This table explains the definition of each field.
+
+| Data Field | Definition |
+| id | The outbound configuration profile id |
+| name |  The name of the outbound configuration template |
+| lookbackPeriodDay | A numeric value representing the skill lookback back period in days, example: 30min value: 0 Max value: 30 |
+| lookbackPeriodHour | A numeric value representing the lookback back period in hours, example: 0 min value:0 Max value: 23
```

Comment:
Optional (random spot): Look into marking up code as code, via backticks. All the data fields look like candidates dor code markdown.